### DISCUSSION TITLE
Issue with setting up the panel

### DISCUSSION THREAD

**Discussion by jovanyeo** [Blackymas/NSPanel_HA_Blueprint#567] [Q&A]
Hi Expert,

I have issue to upload nspanel_eu.tft to the panel, it takes ages to upload and it failed after sometimes (system data error)

Is doing fine with nspanel_us.tft but having orientation problems.

![SmartSelect_20230330_085739_Discord](https://user-images.githubusercontent.com/53246953/228700979-ec40a24d-42ca-4953-a6b5-494b7a0ce333.jpg)

**Comment by edwardtfn** [comment]
Have you ever solved this or are you still with the incorrect orientation?

**Comment by jovanyeo** [comment]
Hi Edward,

Yes, after "N" time of re-flashing the module..

Regards,
Jovan

**Reply by danielmessem** [reply]
Had a similar issue, eventually soldered the wires back and flashed from laptop, worked perfectly after that
